dashdotdb/controllers/serviceslometrics.py

```diff
@@ -1,12 +1,23 @@
 from dashdotdb.services.serviceslometrics import ServiceSLOMetrics
+from dashdotdb.services.serviceslometrics import ServiceSLOMetricsInput
 
 
-def post(user, name, body):
-    slo = ServiceSLOMetrics(name=name)
+def post(user, name: str, body):
+    input_props = ServiceSLOMetricsInput()
+    input_props.name = name
+    slo = ServiceSLOMetrics(input_props=input_props)
     slo.insert(token=user, slo=body)
     return 'ok'
 
 
-def search(cluster, namespace, sli_type, name):
-    slo = ServiceSLOMetrics(cluster, namespace, sli_type, name)
+def search(cluster: str = None, namespace: str = None,
```

Comment:
Just as a FYI, it doesn't seem like Mypy has been enabled on this repo, but our default config would complain about this. It should be `Optional[str]` anytime the default value is `None`. We could enable Mypy on this repo pretty quickly if there isn't a reason not to.

Reply:
>We could enable Mypy on this repo pretty quickly if there isn't a reason not to.

I think that should be fine. I can try to look at how to do that.